web/controllers/account_controller.ex

```diff
@@ -0,0 +1,32 @@
+defmodule ResearchResource.AccountController do
+  use ResearchResource.Web, :controller
+  alias ResearchResource.Redcap.RedcapHelpers
+
+  @redcap_api Application.get_env(:research_resource, :redcap_api)
+
+  plug :authenticate_user when action in [:index]
+
+  def index(conn, _params) do
+    user_details = Enum.to_list @redcap_api.get_user_data(conn.assigns.current_user.ttrrid)
+    # Turn user data from a map to a keyword list
+    |> Enum.map(fn {key, val} -> {String.to_atom(key), val} end)
```

Comment:
I'm fine if you don't want to implement this, but my personal preference would be:
```elixir
user_details =
  conn.assigns.current_user.ttrrid
  |> @redcap_api.get_user_data
  |> Enum.to_list
  |> Enum.map(fn {key, val} -> {String.to_atom(key), val} end)
```

Just so it is consistent how we are applying the functions

But if you would prefer to leave this, that's good with me